src/metabase/channel/impl/slack.clj

```diff
@@ -158,12 +172,19 @@
                                 :text (truncate (:name dashboard) header-text-limit)
                                 :emoji true}}
         link-section    {:type "section"
-                         :fields [{:type "mrkdwn"
-                                   :text (mkdwn-link-text
-                                          (urls/dashboard-url (:id dashboard) parameters)
-                                          (format "*Sent from %s by %s*"
-                                                  (appearance/site-name)
-                                                  creator-name))}]}
+                         :fields (cond->
```

Comment:
nit: the first argument of the `cond->` should be on the same line
```clojure
(cond -> []
  pred1
  body1)
```

Reply:
Ahhh, stawp it, you! 🙈